gradle-palantir-java-format/build.gradle

```diff
@@ -74,15 +74,38 @@ configurations {
         canBeConsumed = false
         canBeResolved = true
     }
+
+    formatterNativeImage {
+        canBeConsumed = false
+        canBeResolved = true
+    }
+
+
 }
 
 dependencies {
     impl project(':palantir-java-format')
+    formatterNativeImage(project(':palantir-java-format-native')) {
+        attributes {
+            attribute(LibraryElements.LIBRARY_ELEMENTS_ATTRIBUTE, project.objects.named(LibraryElements, 'nativeImage'))
+        }
+    }
+
+}
+
+tasks.register("copyNativeImage", Copy.class) {
+    from(configurations.formatterNativeImage)
+    rename { fileName ->
+        String.format("%s.bin", fileName)
+    }
+    into("$buildDir/nativeImage")
 }
 
 def writeImplClasspath = tasks.register("writeImplClasspath") {
+    inputs.files(tasks.named("copyNativeImage"))
     doLast {
         file("$buildDir/impl.classpath").text = configurations.impl.asPath
+        file("$buildDir/nativeImage.path").text = tasks.named("copyNativeImage").get().getOutputs().getFiles().getSingleFile().listFiles()[0].getAbsolutePath()
```

Comment:
required by: https://github.com/palantir/palantir-java-format/pull/1200/files#diff-730f59a46b5a223a716041a78825ed9f9a88c8dbef24bd8c5a07fb2a72adcd22R36. The native image file needs to specify ".bin" extension (this is the artifactType we expect for palantirjavaFormatNative configuration)